src/test/cpp/compare/out/winrt/Component.Composable.h

```diff
@@ -0,0 +1,107 @@
+﻿// WARNING: Please don't edit this file. It was generated by C++/WinRT v2.0.000000.0
+#pragma once
+#include "winrt/impl/Component.Composable.2.h"
+#include "winrt/Component.h"
+namespace winrt::impl
+{
+    template <typename D> hstring consume_Component_Composable_IBase<D>::BaseMethod() const
+    {
+        void* result;
+        check_hresult(WINRT_SHIM(Component::Composable::IBase)->BaseMethod(&result));
+        return { take_ownership_from_abi, result };
+    }
+    template <typename D> Component::Composable::Base consume_Component_Composable_IBaseFactory<D>::CreateInstance(Windows::Foundation::IInspectable const& baseInterface, Windows::Foundation::IInspectable& innerInterface) const
+    {
+        void* value;
+        check_hresult(WINRT_SHIM(Component::Composable::IBaseFactory)->CreateInstance(get_abi(baseInterface), put_abi(innerInterface), &value));
+        return { take_ownership_from_abi, value };
+    }
+    template <typename D> hstring consume_Component_Composable_IDerived<D>::DerivedMethod() const
+    {
+        void* result;
+        check_hresult(WINRT_SHIM(Component::Composable::IDerived)->DerivedMethod(&result));
+        return { take_ownership_from_abi, result };
+    }
+    template <typename D>
+    struct produce<D, Component::Composable::IBase> : produce_base<D, Component::Composable::IBase>
+    {
+        int32_t WINRT_CALL BaseMethod(void** result) noexcept final
+        {
+            try
+            {
+                *result = nullptr;
+                typename D::abi_guard guard(this->shim());
+                *result = detach_from<hstring>(this->shim().BaseMethod());
+                return 0;
+            }
+            catch (...) { return to_hresult(); }
+        }
+    };
+    template <typename D>
+    struct produce<D, Component::Composable::IBaseFactory> : produce_base<D, Component::Composable::IBaseFactory>
+    {
+        int32_t WINRT_CALL CreateInstance(void* baseInterface, void** innerInterface, void** value) noexcept final
+        {
+            try
+            {
+                if (innerInterface) *innerInterface = nullptr;
+                Windows::Foundation::IInspectable winrt_impl_innerInterface;
+                *value = nullptr;
+                typename D::abi_guard guard(this->shim());
+                *value = detach_from<Component::Composable::Base>(this->shim().CreateInstance(*reinterpret_cast<Windows::Foundation::IInspectable const*>(&baseInterface), winrt_impl_innerInterface));
+            if (innerInterface) *innerInterface = detach_abi(winrt_impl_innerInterface);
```

Comment:
Formatting

Reply:
Yep, I need to do a formatting sweep. So glad we have the generated output to visually inspect. 😉